client/me/purchases/remove-purchase/index.jsx

```diff
@@ -343,7 +345,12 @@ const RemovePurchase = React.createClass( {
 					className="remove-purchase__dialog"
 					isVisible={ this.state.isDialogVisible }
 					onClose={ this.closeDialog }>
-					<FormSectionHeading>{ this.translate( 'Remove %(productName)s', { args: { productName } } ) }</FormSectionHeading>
+					<FormSectionHeading>
+						{ translate( 'Your thoughts are needed.' ) }
+					</FormSectionHeading>
+					<p>
+						{ translate( 'Before you go, please answer a few quick questions to help us improve WordPress.com.' ) }
```

Comment:
This comment shouldn't have happened, since the suggestion is the actual string.